src/transaction/log_manager.c

```diff
@@ -194,6 +194,11 @@ extern INT32 vacuum_Global_oldest_active_blockers_counter;
 #define LOG_TDES_LAST_SYSOP_PARENT_LSA(tdes) (&LOG_TDES_LAST_SYSOP(tdes)->lastparent_lsa)
 #define LOG_TDES_LAST_SYSOP_POSP_LSA(tdes) (&LOG_TDES_LAST_SYSOP(tdes)->posp_lsa)
 
+/* temporary, to be removed --> */
+#define log_if_rvdk_format(rvidx, ...) \
```

Comment:
Do you want to keep the stuffs?

Reply:
No, I will remove it before committing patch